src/controllers/simplified-account/settings/worldpay-details/recurring-customer-initiated-credentials/recurring-customer-initiated-credentials.controller.test.js

```diff
@@ -24,6 +24,8 @@ const gatewayAccount = new GatewayAccount({
     credentials: {}
   }]
 })
+const worldpayTasks = new WorldpayTasks(gatewayAccount, SERVICE_ID)
+WorldpayTasks.recalculate = () => { return worldpayTasks }
```

Comment:
have you done it this way because it's not possible to mock the static method?

Reply:
It works when the test is run on its own, but in the context of running all the test (i.e. `npm run test`) it [causes an error as detailed here](https://gds.slack.com/archives/C03B49R80NA/p1739809919389169).